src/tests/index.js

```diff
@@ -0,0 +1,262 @@
+/*global describe, it, beforeEach, before*/
+'use strict'
+const assert = require('assert')
+const Promise = require('bluebird')
+const ServiceManager = require('../lib/service-manager')
+
+const notarySecretKey = 'ZVx0mSsB4+pNdP2q0qq2wEyHKBtoiw2co05Db0sD2Ao6a6skoSHJHAhS5uCKkR8bCu/t3IqKI0jsd3tfbndMcQ=='
+const notaryPublicKey = 'OmurJKEhyRwIUubgipEfGwrv7dyKiiNI7Hd7X253THE='
+
+const services = new ServiceManager(process.cwd())
+
+before(function * () {
+  yield services.startLedger('ledger1', 3001)
+  yield services.startLedger('ledger2', 3002)
+  yield services.startLedger('ledger3', 3003)
+
+  yield services.startConnector('mark', 4001, {
+    pairs: [
+      ['USD@http://localhost:3001', 'USD@http://localhost:3002'],
+      ['USD@http://localhost:3002', 'USD@http://localhost:3001']
+    ],
+    credentials: {
+      'http://localhost:3001': makeCredentials('http://localhost:3001', 'mark'),
+      'http://localhost:3002': makeCredentials('http://localhost:3002', 'mark')
+    }
+  })
+
+  yield services.startConnector('mary', 4002, {
+    pairs: [
+      ['USD@http://localhost:3003', 'USD@http://localhost:3002'],
+      ['USD@http://localhost:3002', 'USD@http://localhost:3003']
+    ],
+    credentials: {
+      'http://localhost:3002': makeCredentials('http://localhost:3002', 'mary'),
+      'http://localhost:3003': makeCredentials('http://localhost:3003', 'mary')
+    }
+  })
+
+  yield services.startNotary('notary1', 6001, {
+    secretKey: notarySecretKey,
+    publicKey: notaryPublicKey
+  })
+})
+
+// Run this before each test
+beforeEach(function * () {
```

Comment:
No, I think that's probably more trouble than its worth. Each test run resets the full state. If we need more granularity than that (right now we don't) we could just have each test use a different set of accounts, rather than having them all use alice & bob.